app/models/installed_technology.rb

```diff
@@ -27,7 +28,7 @@ def self.template
   #
   # Returns true or false.
   def exists?
-    type.blank? || type == 'generic'.freeze || Technology.exists?(key: type)
+    type.blank? || type =~ /generic|base_load/ || Technology.exists?(key: type)
```

Comment:
This regex – and the same one in `Technology` – seems too permissive; it would match *any* technology with "generic" or "base_load" anywhere the key. I suggest changing it to `\A(?:generic|base_load)\Z`. Maybe make it a constant on `Technology` so that it doesn't need to be repeated?